overlay.d/07el9/etc/ssh/sshd_config.d/41-rhcos-defaults.conf

```diff
@@ -0,0 +1,8 @@
+# Disable PasswordAuthentication in SSH to preserve RHCOS 8 defaults
+# See: https://issues.redhat.com/browse/OCPBUGS-11613
+# See: https://github.com/openshift/os/issues/1216
+PasswordAuthentication no
+
+# Enable ClientAliveInterval and set to 180
+# See: https://bugzilla.redhat.com/show_bug.cgi?id=1701050
+ClientAliveInterval 180
```

Comment:
Aside: might be good to check whether this is still needed in el9. E.g. we don't have this in FCOS and no one has complained AFAIK. But if it's an issue there too, we should upstream it to FCOS. If not, then we can drop it here too.

Not a blocker.